src/cpu/x64/jit_avx512_core_amx_conv_kernel.hpp

```diff
@@ -473,22 +474,20 @@ struct jit_avx512_core_amx_bwd_data_kernel_t : public jit_generator {
         : jit_generator(jit_name(), avx512_core_amx)
         , jcp(ajcp)
         , attr_(attr)
-        , eltwise_injector_(nullptr) {
+        , eltwise_injector_(nullptr)
+        , bwd_data_copy_kernel_(nullptr) {
```

Comment:
this is interesting. not quite a bug, but it looks like they forgot the member variable as part of this initializer.